pkg/config/cloud_lb_config_test.go

```diff
@@ -0,0 +1,55 @@
+package config
+
+import (
+	"net"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var (
+	testKubeconfigPath    = "/test/path/kubeconfig"
+	testClusterConfigPath = "/test/path/clusterConfig"
+	testResolvConfPath    = "/test/path/resolvConf"
+	testApiLBIPv4         = net.ParseIP("192.168.0.111")
+	testApiIntLBIPv4      = net.ParseIP("10.10.10.20")
+	testIngressOneIPv4    = net.ParseIP("192.168.20.140")
+	testIngressTwoIPv4    = net.ParseIP("10.10.10.40")
+	testClusterLBConfig   = ClusterLBConfig{
+		ApiLBIPs:     []net.IP{testApiLBIPv4},
+		ApiIntLBIPs:  []net.IP{testApiIntLBIPv4},
+		IngressLBIPs: []net.IP{testIngressOneIPv4, testIngressTwoIPv4}}
+	expectedApiLBIPv4      = "192.168.0.111"
+	expectedApiIntLBIPv4   = "10.10.10.20"
+	expectedIngressOneIPv4 = "192.168.20.140"
+	expectedIngressTwoIPv4 = "10.10.10.40"
+	updateNode             = Node{}
```

Comment:
No need for those 2 to be global as they do not carry any state that needs to be shared